Automattic/newspack-plugin
# Remove performance wizard

### All Submissions:






### Changes proposed in this Pull Request:

The Performance Wizard doesn't add much value to the Newspack Plugin anymore. PWA options are automatically set up for users and the site icon can be accessed via Site Design > Settings > Advanced settings > Site Identity

### How to test the changes in this Pull Request:

1. Check that you don't see the Performance Wizard in the Sidebar and the Dashboard
2. Check that PWA isn't broken
3. Check that you can still edit the site icon via the Customizer

### Other information:
